clients/python/client/osparc/_http_client.py

```diff
@@ -8,16 +8,9 @@ class AsyncHttpClient:
     """Async http client context manager"""
 
     def __init__(
-        self,
-        *,
-        base_url: httpx.URL,
-        request_type: Optional[str] = None,
-        url: Optional[str] = None,
-        auth: Optional[httpx.BasicAuth] = None,
+        self, *, request_type: Optional[str] = None, url: Optional[str] = None, **kwargs
```

Comment:
Q: the client needs to work for 3.6, right?  we should add at some point `py-upgrade` pre-commit hooks

Reply:
MINOR: I would rename `kwargs` by `httpx_async_client_kwargs`

Reply:
good point, I have fixed this